packages/lisk-cryptography/test/nacl/nacl.ts

```
@@ -195,6 +195,21 @@ describe('nacl', () => {
 						Buffer.from(decryptedMessageBytes).toString('utf8'),
 					).to.be.eql(defaultMessage);
 				});
+
+				it('should throw an error with invalid message', () => {
```

`should throw an error *for* invalid message` is better.